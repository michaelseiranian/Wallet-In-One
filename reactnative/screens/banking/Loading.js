import { View, ActivityIndicator } from 'react-native';
import { useTheme } from 'reactnative/src/theme/ThemeProvider'

export default function Loading() {

  const {dark, colors, setScheme} = useTheme();

  return (
    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center', backgroundColor: colors.background }}>
        <ActivityIndicator color={colors.primary} size="large" />
    </View>
  );
}
