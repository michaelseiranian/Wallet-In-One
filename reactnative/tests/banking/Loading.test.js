import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../screens/banking/Loading'

describe('<HomePage />', () => {
    it('snapshot test', () => {
        const snapshot = renderer.create(<Loading />);
        expect(snapshot).toMatchSnapshot();         
      });
  });