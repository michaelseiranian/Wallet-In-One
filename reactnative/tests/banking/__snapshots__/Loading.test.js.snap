// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> snapshot test 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    color="purple"
    size="large"
  />
</View>
`;
